build.gradle

```diff
@@ -10,6 +10,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.novoda:bintray-release:0.8.0'
```

Comment:
Are we realising any binaries - do we need this (at the moment)?

Reply:
Not really, but it was needed in order to keep the source code of possumgather and possumcore intact. Removed the dependency and just removed it from possumgather as well